thorlcr/master/thgraphmanager.cpp

```diff
@@ -850,7 +850,22 @@ void CJobManager::reply(IConstWorkUnit *workunit, const char *wuid, IException *
     if (e)
     {
         if (!exitException)
+        {
             exitException.setown(e);
+            Owned<IWorkUnit> w = &workunit->lock();
+            if (e)
```

Comment:
Not sure what the point of this test is - it's been tested above

Reply:
agreed, now removed.